src/resources/faq.md

```diff
@@ -131,22 +147,33 @@ Flutter includes:
 * APIs for unit and integration tests
 * Interop and plugin APIs to connect to the system and 3rd-party SDKs
 * Headless test runner for running tests on Windows, Linux, and Mac
-* [Dart DevTools][] for testing, debugging, and profiling your app
+* [Flutter DevTools][] (also called Dart DevTools)
+  for testing, debugging, and profiling your app
 * Command-line tools for creating, building, testing, and
   compiling your apps
 
+[Dart DevTools]: {{site.url}}/tools/devtools
+
 ### Does Flutter work with any editors or IDEs?
 
-We provide plugins for [Android Studio][],
-[IntelliJ IDEA][], and [VS Code][]. See
-[editor configuration][] for setup details,
-and [Android Studio/IntelliJ][] and [VS Code][]
+We provide plugins for [VS Code][],
+[Android Studio][], and [IntelliJ IDEA][].
+See [editor configuration][] for setup details,
+and [VS Code][] and [Android Studio/IntelliJ][]
 for tips on how to use the plugins.
 
 Alternatively, you can use the `flutter` command
 from a terminal, along with one
 of the many editors that support [editing Dart][].
 
+
```

Comment:
nitty-nit: throughout this document, the list of links is sometimes separated by two blank lines, sometimes by one. Not sure if you want to follow a unified style here?

Reply:
LOL. Yeah, if there is only one link, I tend to use one line. If there are multiple links, I tend to use two. I am a bit conflicted about it.  :)